sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/PathPattern.java

```diff
@@ -25,10 +25,18 @@
 import org.apache.commons.lang.StringUtils;
 import org.sonar.api.utils.PathUtils;
 import org.sonar.api.utils.WildcardPattern;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
 
 @ThreadSafe
 public abstract class PathPattern {
 
+  private static final Logger LOG = Loggers.get(PathPattern.class);
+
+  /**
+   * @deprecated since 6.6
+   */
+  @Deprecated
   private static final String ABSOLUTE_PATH_PATTERN_PREFIX = "file:";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)